site/docs/encoding.md

```diff
@@ -180,27 +180,27 @@ In addition to the constant `value`, [value definitions](#value-def) of mark pro
 See [the `condition`](condition.html) page for examples how to specify condition logic.
 
 {:#text}
-## Text and Tooltip Channels
+## Text, Tooltip, and HREF Channels
```

Comment:
href should be on its own category -- it doesn't have `format` property

Reply:
Hmm, I was thinking that one might want to format a number to be a URL like `www.site.com?page=${id}`.

Reply:
Yeah but having `format` wouldn't help you do that anyway?

https://github.com/d3/d3-format#locale_format

Reply:
oh I see -- you can use "fill"

Reply:
Actually you can't .. fill is for a slightly different purpose

Reply:
You're right. I was thinking of dates where you can do something like `foo %H bar`